File: daily-coding/coding-bat/python/array123.py
# Given an array of ints, return True if the sequence of numbers 1, 2, 3 appears in the array somewhere.

def array123(nums):
  oneTwoThree = [1,2,3]
  search = []
  match = False
  for x in range(0,len(nums)-2):
    search = [nums[x],nums[x+1],nums[x+2]]
    if search == oneTwoThree:
      match = True

  return match

File: daily-coding/coding-bat/java/array123.java
// Given an array of ints, return true if the sequence of numbers 1, 2, 3 appears in the array somewhere.

public boolean array123(int[] nums) {
  int length = nums.length;
  boolean match = false;
  int sum = 0;

  for(int i = 0; i < length - 2;i++){
    sum = nums[i] + nums[i+1] + nums[i+2];
    if(sum == 6){
      match = true;
    }
  }
  return match;
}
